specs/system_config.md

```diff
@@ -111,3 +115,32 @@ The contained log events are filtered and processed as follows:
 Note that individual derivation stages may be processing different L1 blocks,
 and should thus maintain individual system configuration copies,
 and apply the event-based changes as the stage traverses to the next L1 block.
+
+## Ecotone Activation
+
+### Constants
+
+The initial fee paramaters for the Ecotone hardfork are noted in the following table.
+
+| `l1BasefeeScalar` | `l1BlobBasefeeScalar` |
+|---------------------------------------------------|---|
+| Value of `scalar` at Ecotone activation block - 1 | 0 |
+
+On the Ecotone hardfork activation block, the L2 view of the `SystemConfig` should
+migrate the current value of the `scalar` to the `l1BasefeeScalar` and set the
+`l1BlobBasefeeScalar` value to 0. The `scalar` is defined to be a `uint256` while
+the `l1BasefeeScalar` is defined to be a `uint32`, so any `scalar` value that is larger
+than `type(uint32).max` at time of conversion should be migrated as `type(uint32).max`.
+
+The `SystemConfig` contract MUST set these values as the initial values to guarantee
+the L2's view of the `SystemConfig` remains consistent.
+
+Any `ConfigUpdate` events for the Ecotone fee parameters before MAY be parsed or
+MAY be ignored by the derivation pipeline. Since the values are overwritten on
+the hardfork activation block and are not consumed until the hardfork activation block
+height + 1 , there is no functional difference between the two approaches.
```

Comment:
This is interesting, but not entirely clear to me:

> Since the values are overwritten on the hardfork activation block and are not consumed until the hardfork activation block height + 1

Why does the hardfork activation block affect the values in the `SystemConfig` contract?
What is consuming the values, and why is it in the activation block + 1?

Reply:
The activation doesn't impact the values in the contract itself - it impacts the L2 view of the contract